test/run-integration.js

```diff
@@ -137,6 +138,20 @@ function setSettings(t) {
     .then(noop, _.bind(t.error, t));
 }
 
+function getSettings(t) {
+  t.plan(2);
+
+  index
+    .setSettings({attributesForFaceting: ['searchable(category)']})
+    .then(function() {return index.getSettings({advanced: 1});})
+    .then(get('attributesForFaceting'))
+    .then(_.partialRight(t.deepEqual, ['searchable(category)'], 'Settings were get (advanced)'))
+    .then(index.getSettings)
```

Comment:
Here getSettings is called with one argument (result from previous Promise, even `undefined` wdyt?

Reply:
good call, amending the test to be really with no arguments